src/main.rs

```diff
@@ -494,7 +494,7 @@ fn run() -> Result<(), failure::Error> {
             failure::bail!("You can only pass --env or --release, not both")
         }
         let manifest = settings::target::Manifest::new(config_path)?;
-        let target;
+        let mut target;
```

Comment:
target must be mutable in order to update its kv-namespaces for publish. could alternatively write a separate constructor that did this inside target, but for now this works Just Fine.